thirdeye/thirdeye-frontend/app/pods/manage/alert/explore/route.js

```diff
@@ -129,14 +129,16 @@ const processRangeParams = (bucketUnit, duration, start, end) => {
  * Setup for query param behavior
  */
 const queryParamsConfig = {
+  refreshModel: true,
```

Comment:
I know we went back and forth regarding this. Just want to make sure what's the latest consensus here.

Reply:
Yes - I realize I do need this to cause a reload when I'm ready to check for new anomalies.